perfkitbenchmarker/providers/profitbricks/flags.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from perfkitbenchmarker import flags
+
+
+# Locations
+US_LAS = 'us/las'
+DE_FKB = 'de/fkb'
+DE_FRA = 'de/fra'
+
+# Zones
+ZONE_1 = 'ZONE_1'
+ZONE_2 = 'ZONE_2'
+
+flags.DEFINE_string('profitbricks_config',
+                    os.getenv('PROFITBRICKS_CONFIG',
+                              '~/.config/profitbricks-auth.cfg'),
+                    ('Path to config file containing your email and password. '
+                     'Can also be set via $PROFITBRICKS_CONFIG environment '
+                     "variable.\n(File format: email:password)"))
+
+flags.DEFINE_enum('location',
```

Comment:
I originally created the `--profitbricks_zone` flag because when I tried to run a benchmark using the `--zone` flag I received the below error.
```
$ ./pkb.py --cloud=ProfitBricks --benchmarks=ping --zone=ZONE_2
ERROR    Unknown command line flag 'zone'. Did you mean: zones?
Usage: ./pkb.py ARGS
```

If running the same benchmark using the `--zones` flag, everything works perfectly.  Please advise if this behavior is expected?
```
$ ./pkb.py --cloud=ProfitBricks --benchmarks=ping --zones=ZONE_2
2016-06-27 23:59:26,569 44adf054 MainThread INFO     Verbose logging to: /var/folders/8f/2q6dw27d2tsbhzf41vmt3gv80000gn/T/perfkitbenchmarker/runs/44adf054/pkb.log
2016-06-27 23:59:26,570 44adf054 MainThread INFO     PerfKitBenchmarker version: v1.4.0-103-ge78e513
2016-06-27 23:59:26,570 44adf054 MainThread INFO     Flag values:
--cloud=ProfitBricks
--zones=ZONE_2
--benchmarks=ping

[...]
-------------------------PerfKitBenchmarker Complete Results-------------------------
{'metadata': {'perfkitbenchmarker_version': 'v1.4.0-103-ge78e513',
              'vm_1_cloud': 'ProfitBricks',
              'vm_1_image': u'2a73539c-27ba-11e6-8e88-52540005ab80',
              'vm_1_machine_type': 'Small',
              'vm_1_vm_count': 1,
              'vm_1_zone': 'ZONE_2'
```
  
Combining the locations/zones into a single parameter is doable.  My only concern is that this would be unnatural for ProfitBricks users.  With the ProfitBricks platform you pass your location and availability zone arguments separately, as opposed to the AWS way.  If we did combine them, something like `us/las-zone_1` and `us/las-zone_2` should work fine.  Thoughts?